plone/app/content/utils.py

```diff
@@ -9,6 +11,11 @@ def custom_json_handler(obj):
         return None
     if type(obj) in (datetime.datetime, datetime.date):
         return obj.isoformat()
+    if type(obj) == DateTime:
```

Comment:
Btw. - I know you did it the same as one line above - is there any reason to compare the type instead of using ``isinstance(obj, DateTime)``?